specification/hdinsight/resource-manager/readme.md

```diff
@@ -32,6 +32,23 @@ azure-arm: true
 tag: package-2018-06-preview
 ```
 
+## Suppression
+ ``` yaml
+ directive:
+   - suppress: DefinitionsPropertiesNamesCamelCase
+     reason: This would require a breaking change, and the capabilities API was removed in version 2018-06-01-preview.
+     from: locations.json

```

Comment:
please change this location to the specific version path - `Microsoft.HDInsight/preview/2015-03-01-1/preview/locations.json` . That way linter should be suppressed only in the old version.

Reply:
Awesome, thanks!